telemetry/csharp/AwsToolkit.Telemetry.Events/AwsToolkit.Telemetry.Events.csproj

```diff
@@ -16,7 +20,11 @@
 	</PropertyGroup>
 	<ItemGroup>
 	  <PackageReference Include="log4net" Version="2.0.8" />
+	  <PackageReference Include="NuGet.Build.Tasks.Pack" Version="5.2.0"/>
 	</ItemGroup>
+	<Target Name="PreBuild" BeforeTargets="PreBuildEvent">
```

Comment:
Can you update the "AwsToolkit.Telemetry.Events" section of "telemetry/csharp/README.md" with the command (mentioned in the pr description) that is used to produce a nuget package.

Additionally, you might want to include a release configuration in there:
```
msbuild AwsToolkit.Telemetry.Events\AwsToolkit.Telemetry.Events.csproj /t:Build /p:Version=$(VERSION) /p:Configuration=Release
```